lib/src/widgets/buttons.dart

```diff
@@ -392,3 +396,85 @@ class _CardButtonState extends State<CardButton> {
     }
   }
 }
+
+/// Button to add onTap on long pressing
+///
+/// ### Note
+/// Widgets with `tooltip` don't handle onLongPress
+/// Child's onTap will have priority
+class LongPressButton extends StatefulWidget {
+  const LongPressButton({
+    required this.callback,
+    required this.child,
+    this.longDelay = const Duration(milliseconds: 600),
```

Comment:
If I got it well the 3 three delays are there to produce an acceleration effect. So instead of `longDelay`, `middleDelay` and `minDelay` names, which are not very clear I think it'd be better to use a list of duration param:
```dart
/// Delays between callbacks at the beginning. Leave default to get an acceleration effect.
final List<Duration> triggerDelays;

/// Delay between callbacks on hold.
final Duration holdDelay
```

Reply:
Although making it into a list is great but we don't know the length of the list. Maybe adding an asset such that the length of the list is less than 3 would be ideal.

Reply:
oh yeah, good catch, but is a variable length really problematic?

or instead of a list a 3-uple (Duration, Duration, Duration) is possible too.

Reply:
Not really but a long list might be so capping it is a good idea. Also it can be empty if we don't want the acceleration effects. 

I already added an assert, i think that should be enough